src/lib/rest/RestService.cpp

```diff
@@ -149,7 +149,7 @@ std::string payloadParse
 *
 * tenantCheck - 
 */
-static std::string tenantCheck(const std::string& tenant)
+extern std::string tenantCheck(const std::string& tenant)
```

Comment:
Yeah, 'extern' is not needed. I left it to be more clear.
Look at the issue #2781. All the info is there.
Enough, right?